### PR TITLE
[RS-688] Suggest forecasting v1

### PR DESCRIPTION
**Pull request by skahmann3**
Checklist for reviewer:

Need to add schema

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

**Comment by skahmann3** [comment]
Kicking off forecasting revamp tickets. Will need to determine whether I want to wait for KPI forecasts to be done by country (which could simplify this query) or move forward as is.

File: sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_forecasting_v1/query.sql [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+-- DAU SHARE BY COUNTRY
+-- Used to scale automated global KPI forecasts down to US clients only
+WITH client_counts AS (
+  SELECT
+    country,
+    submission_date,
+    COUNT(*) AS total_clients,
+    COUNT(
+      CASE
+  -- SUGGEST DESKTOP ELIGIBILITY REQUIREMENTS
```
Comment (rebecca-burwei) [inline sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_forecasting_v1/query.sql:13]: These are the Suggest eligibility requirements: https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry_derived/suggest_clients_daily_v1/query.sql#L144

The 2 filters on version are to select versions >= 92.0.1

File: sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_forecasting_v1/query.sql [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    AND active_hours_sum > 0
+    AND uri_count > 0
+  GROUP BY
+    country,
+    submission_date
+),
+grand_total AS (
+  SELECT
+    submission_date,
+    SUM(total_clients) AS monthly_total
```
Comment (rebecca-burwei) [inline sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_forecasting_v1/query.sql:43]: This calculates cumulative days of use over the entire month, not the number of unique clients active in a month. Is this what you want?

Comment (rebecca-burwei) [reply sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_forecasting_v1/query.sql:43]: oh, nevermind, I misread. It is DAU. However, the name "monthly_total" is confusing.

**Review by rebecca-burwei — APPROVED** [review]
This looks great. At some point, we'll have to convert this all over to use the new Glean urlbar events (and Glean client counts), but it doesn't have to happen now.

**Comment by skahmann3** [comment]
@alekhyamoz and @curtismorales - here's the second PR related to MBR work. Will one of you please support?

**Pull request merged**